var/spack/repos/builtin/packages/py-astropy-helpers/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAstropyHelpers(PythonPackage):
+    """The astropy-helpers package includes many build,
+    installation, and documentation-related tools used by the
+    Astropy project, but packaged separately for use by other
+    projects that wish to leverage this work."""
+
+    homepage = "https://github.com/astropy/astropy-helpers"
+    url      = "https://github.com/astropy/astropy-helpers/archive/v4.0.1.tar.gz"
+
+    version('4.0.1',  sha256='88602971c3b63d6aaa6074d013f995d1e234acb3d517d70d7fcebd30cdaf5c89')
+    version('4.0rc1', sha256='322aeda0ff0c8459efc6306b4d26eb115d09e96946f48f1dfe6e91f41ea6af1a')
```

Comment:
Is there any reason to add the release candidates? We normally don't add alpha/beta/rc versions unless someone really needs them, or the release hasn't come out yet. Honestly, I usually only add the latest version since I don't want to have to support older versions.

Reply:
Cleaned up old versions as requested.